DESIGN-NOTES.md

```diff
@@ -38,11 +38,11 @@ Instrumentation Engine allows multiple profilers co-exist in the same process. I
 
 One of the goals of this design is to allow for consumers of the new instrumentation engine to be implemented in separate dlls. This allows them to be implemented by separate teams, and shared between different products. Achieving these goals will require a binary compatibility contract. Instrumentation Engine uses a form of Reg-Free COM that reuses the interface concept including IUnknown but does not bring in com baggage such as apartment threading or marshaling. This nicely matches the existing model of ICorProfiler.
 
-Sticking with the Intellitrace terminology, these instrumentation components called "Instrumentation Methods" which will essentially be free threaded com objects loaded by direct call to the class factory using the com server exports (DllGetClassObject etc…). This has the advantage of requiring a simple vtable dispatch for each cross component call keeping overhead to a minimum.
+Sticking with the IntelliTrace terminology, these instrumentation components called "Instrumentation Methods" which will essentially be free threaded com objects loaded by direct call to the class factory using the com server exports (DllGetClassObject etc…). This has the advantage of requiring a simple vtable dispatch for each cross component call keeping overhead to a minimum.
```

Comment:
>IntelliTrace [](start = 18, length = 12)

Sorry, I never looked at the initial check-in. Why is it that we mention IntelliTrace in this so much? Seems like a historical detail that isn't much use to people who are trying to write instrumentation engines.